src/Features/Renderer.cpp

```diff
@@ -874,7 +878,8 @@ static void startRender()
 
     // Busy-wait until the rendering has started so that we don't miss
     // any frames
-    while (!g_render.isRendering.load() && !g_render.workerFailedToStart.load()) return;
+    while (!g_render.isRendering.load() && !g_render.workerFailedToStart.load())
+        continue;
```

Comment:
Not really any point using continue here, just do `while (!g_render.isRendering.load() && !g_render.workerFailedToStart.load());` - that's what I've done elsewhere in the file (I can only assume it was 2am when I wrote this broken line or something lol) so it's more consistent then